app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,14 @@
+class ReviewSerializer < BaseSerializer
+  type :review
+
+ 	attributes(
+		:id,
+		:user_id,
+		:item_id,
+    :body,
+		:rating,
+		:created_at,
+		:updated_at
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/IndentationStyle: Tab detected in indentation.